types/application.d.ts

```diff
@@ -72,3 +83,21 @@ export const appLanguage: string;
  * User's OS-wide locale setting. May not match the XD UI, since XD does not support all world languages. Includes both language and region (e.g. "fr_CA" or "en_US").
  */
 export const systemLocale: string;
+
+/**
+ * Information about the document which this instance of the plugin is attached to.
+ *
+ * > **Tip**
+ * >
+ * > _This does **not** indicate the frontmost "active" document window in the XD application._
+ * >
+ * > In XD, each document window loads a separate copy of your plugin. When a given instance of your plugin calls this API, you will always receive information about the document that this instance of the plugin is attached to, even if it's not the active window.
+ *
+ * @example ```js
+let application = require("application");
+let documentInfo = application.activeDocument;
+console.log("Document title: " + documentInfo.name);
+console.log("Document ID: " + documentInfo.guid);
+ ```
+ */
```

Comment:
While this _**does**_ render correctly in VSCode for some reason, it would be best to get it inline with the rest. I have removed the `js` annotation for two reasons:

1. My previous markup suggestions also elided it.
1. This JavaScript is also valid TypeScript (and vice versa) so it shouldn't matter.

```suggestion
 * @example 
 * ```
 * let application = require("application");
 * let documentInfo = application.activeDocument;
 * console.log("Document title: " + documentInfo.name);
 * console.log("Document ID: " + documentInfo.guid);
 * ```
 */
```

Reply:
For whatever reason, the best way I've found for including examples is to remove the backticks entirely (tested in VSCode and WebStorm)... I'll do that, for now.